cv_bridge/CMakeLists.txt

```diff
@@ -46,10 +46,9 @@ add_subdirectory(src)
 # This is overwritten each loop, but which one it points to doesn't really matter.
 set(cv_bridge_lib_dir "$<TARGET_FILE_DIR:${PROJECT_NAME}>")
 
-find_package(ament_lint_auto REQUIRED)
-ament_lint_auto_find_test_dependencies()
-
 if(BUILD_TESTING)
+  find_package(ament_lint_auto REQUIRED)
+  ament_lint_auto_find_test_dependencies()
```

Comment:
This code is already present in the [`test/CMakeLists.txt` file](https://github.com/ros-perception/vision_opencv/blob/ros2/cv_bridge/test/CMakeLists.txt). So having it here as well is redundant, it would be better to remove it.